obs-vscode-extension/server/src/server.ts

```diff
@@ -0,0 +1,80 @@
+import { createConnection, ProposedFeatures, TextDocuments, InitializeParams, TextDocumentSyncKind, TextDocumentChangeEvent, Position, Diagnostic, Range, DiagnosticSeverity } from "vscode-languageserver";
+import { TextDocument } from "vscode-languageserver-textdocument";
+
+import * as process from "child_process";
+import { promises as fs } from "fs";
+
+const connection = createConnection(ProposedFeatures.all);
+const documents: TextDocuments<TextDocument> = new TextDocuments(TextDocument);
+
+connection.onInitialize((params: InitializeParams) => {
+  return {
+    capabilities: {
+      textDocumentSync: TextDocumentSyncKind.Full,
+      // completionProvider: {
```

Comment:
Is this for future use? You should either remove the commented-out code or extend the comment with an explanation.